bitnami/kafka/templates/scripts-configmap.yaml

```diff
@@ -73,6 +73,7 @@ data:
 
     if [[ -f "/bitnami/kafka/data/meta.properties" ]]; then
         export KAFKA_CFG_BROKER_ID="$(grep "broker.id" /bitnami/kafka/data/meta.properties | awk -F '=' '{print $2}')"
+        ID="$((KAFKA_CFG_BROKER_ID - {{ .Values.minBrokerId }}))"
```

Comment:
I don't understand how this can fix the issue... 

Let's say that I install the chart the chart (1s time):
```console
$ helm install kafka bitnami/kafka --set minBrokerId=100 --set replicaCount=2
```
The broker IDs at `/bitnami/kafka/data/meta.properties` are:
```console
$ kubectl exec kafka-0 -- grep "broker.id" /bitnami/kafka/data/meta.properties
broker.id=100
$ kubectl exec kafka-1 -- grep "broker.id" /bitnami/kafka/data/meta.properties
broker.id=101
```
Why I should subtract the "minBrokerId" when restarting the container?